R/kmeans-clustering-method.R

```diff
@@ -0,0 +1,528 @@
+#' Build clusters using kmeans()
+#'
+#' @description This step allows you to use kmeans() to build clusters, based on
+#' your data.
+#' @docType class
+#' @usage KmeansClustering(object, dataType, df, grainCol, labelCol, numOfCluster,
+#' pca, usePrinComp,numOfPrinComp,impute, debug)
+#' @importFrom R6 R6Class
+#' @import ranger
+#' @import cluster
+#' @param object of UnsuperviseModelParameters class for $new() constructor
+#' @param dataType The type of your data set('numeric' for all numeric data set,
```

Comment:
`dataType` param seems unnecessary. Couldn't we just check that all columns are numeric and either

1) Warn them that they need to remove non-numeric cols from their dataset or

2) Let them know that we automatically did this and tell them which ones were removed?